client/src/components/Header.js

```diff
@@ -89,8 +97,7 @@ const activeStyles = {
   fontWeight: "600",
   color: `${colors.royalBlue};`,
 };
-
-export default ({ isAuthenticated, onMenuClick }) => {
+export default ({ isAuthenticated,onFeedbackIconClick, onMenuClick }) => {
```

Comment:
space before `onFeedback` does seems like your linter / formatter isn't working
don't forget to put `client/.env.js : EXTEND_ESLINT=true`

Reply:
I changed this like when I had merged conflict from github. so I kinda missed the place on here. It should be fixed now